falcon/util/misc.py

```diff
@@ -95,7 +100,7 @@ def decorator(func):
     _lru_cache_for_simple_logic = functools.lru_cache  # type: ignore
 
 
-def is_python_func(func):
+def is_python_func(func: Union[Callable, Any, None]) -> bool:
```

Comment:
isn't any enough?

Reply:
Probably `Optional[Callable]` is enough. When can this `func` be `None`?

Reply:
I removed the None type, Any is needed when method is a function.